Rocket.Chat/Helpers/Ask.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Ask.swift
+//  Rocket.Chat
+//
+//  Created by Luca Justin Zimmermann on 25/01/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+protocol Asker: class {
+    func ask(title: String, message: String, buttons: [String], handlers: [((UIAlertAction) -> Void)?])
+}
+
+extension UIViewController: Asker {
+    func ask(title: String, message: String, buttons: [String], handlers: [((UIAlertAction) -> Void)?]) {
+        let ask = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        if buttons.count != handlers.count {
```

Comment:
Maybe a touple instead of two separate parameters would be better? (button: String, handler: ((UIAlertAction) -> Void)?)

Reply:
Good catch @cardoso, changed